src/main/java/com/example/bookstore/config/DateTimeConfig.java

```diff
@@ -0,0 +1,8 @@
+package com.example.bookstore.config;
+
+import java.time.format.DateTimeFormatter;
+
+public class DateTimeConfig {
+    public static final DateTimeFormatter format = DateTimeFormatter
```

Comment:
java code convention violation 
```suggestion
    public static final DateTimeFormatter ***** = DateTimeFormatter
```